src/ClientHandle.cpp

```diff
@@ -1884,6 +1884,18 @@ void cClientHandle::HandleUseItem(bool a_UsedMainHand)
 
 
 
+
+void cClientHandle::HandleResourcePack(UInt8 a_Status)
+{
+	// Kick player if client declined the resource pack
+	if(a_Status == 1)
+		Kick("You must accept the resource pack");
```

Comment:
From the discussion in #3349 , it seems like I need to call a lua hook here. Since I don't know how, I'm leaving this code here for now to signify the gist of what I think the server should do, i.e. kick the client if it rejects the resource pack

Reply:
I did the part needed for the hook if you want, I made a PR on your own branch at https://github.com/plan1231/cuberite/pull/1

Reply:
Thanks tonitch! Sorry I didn't see this earlier.